docs/src/main/sphinx/release/release-406.md

```diff
@@ -85,7 +85,7 @@
 
 * Add support for changing column types. ({issue}`15515`)
 * Add [support for the JDBC catalog](iceberg-jdbc-catalog). ({issue}`9968`)
-* ReduceAdd support for changing fields in columns of row type. latency for `INSERT` queries on unpartitioned tables. ({issue}`15708`)
```

Comment:
is `Add support for changing fields in columns of row type.` somewhere?

Reply:
I think "Add support for changing column types. ({issue}`15515`)" at L86 is sufficient.